src/js/modules/core/components/HomePage.js

```diff
@@ -6,6 +6,8 @@ import injectSheet from "react-jss";
 
 import sections from "../../sections";
 
```

Comment:
no need for newline here. just group the imports in two groups: one for libraries and one for not libraries (selectors, actions, components, etc.)